types/index.d.ts

```diff
@@ -765,12 +765,20 @@ export interface EventComponentProps extends TranslationContextValue {
 
 export interface GalleryProps {
   images: Client.Attachment[];
+  t: i18next.TFunction;
```

Comment:
t is supplied by context and better to leverage `useContext` and have it removed from types